ci/dependabot-updater.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -ex
+cd "$(dirname "$0")/.."
+source ci/_
+
+commit_range="$(git merge-base HEAD origin/master)..HEAD"
+parsed_update_args="$(
+  git log "$commit_range" --author "dependabot-preview" --oneline -n1 |
+    grep -o 'Bump.*$' |
+    sed -r 's/Bump ([^ ]+) from [^ ]+ to ([^ ]+)/-p \1 --precise \2/'
+)"
+package=$(echo "$parsed_update_args" | awk '{print $2}')
+if [[ -n $parsed_update_args ]]; then
+  # shellcheck disable=SC2086
+  _TARGET_LOCK_FILES=$(git grep --files-with-matches "$package" :**/Cargo.lock) \
```

Comment:
Using `_TARGET_LOCK_FILES` to pass a list into `scripts/cargo-for-all-lock-files.sh` feels a little too sneaky to me.  It would be nice to fit this on the `scripts/cargo-for-all-lock-files.sh` command line.

`$ scripts/cargo-for-all-lock-files.sh $(_TARGET_LOCK_FILES) -- update $parsed_update_args` perhaps?  And if there's no `--` argument then we use `$(git ls-files :**/Cargo.lock)`

Reply:
Yeah, I've been concerned a bit too... I followed an easier path considering these script's intended usage frequency. Thanks for accurately pin-pointing this while reviewing! I've changed my mind and give this some love. :)